Source/API/EbApi.h

```diff
@@ -505,7 +505,12 @@ typedef struct EB_H265_ENC_CONFIGURATION
     uint8_t                 fpsInVps;
 
 
-    // Application Specific parameters
+    // VBV Parameters
+
```

Comment:
please use the same format to explain the usage of parameters and default value. EbApi.h is the interface file, a lot of developer refer to this file for more info. Please provide more info.
/**/

Reply:
Resolved and pushed patch (Commit : 9db1593)